plugins/woocommerce-admin/client/two-column-tasks/sectioned-task-list.scss

```diff
@@ -127,3 +127,25 @@
 		}
 	}
 }
+
+.woocommerce-setup-panel {
+	h1 {
+		font-weight: 400;
+	}
+
+	.woocommerce-task-progress-header {
```

Comment:
This looks good.  The completed state in the 2 experiments feels a bit odd to me; not sure if this is intentional or not.  cc @jarekmorawski 
<img width="449" alt="Screen Shot 2022-05-24 at 10 00 08 AM" src="https://user-images.githubusercontent.com/10561050/170055013-b18c2fc3-e7fa-4e58-bac0-058c4d106ed0.png">
<img width="457" alt="Screen Shot 2022-05-24 at 9 58 09 AM" src="https://user-images.githubusercontent.com/10561050/170055016-de655f7a-90fa-469d-96db-e2fb892193e1.png">

Reply:
The "Welcome to Site Name" piece looks odd to me here.  The success screen looks good though.

Reply:
Oh, I totally missed the header 🙈 In the design, the header has a different appearance in the side panel; the fonts and paddings are smaller to bring out the progress bar and the tasks. When completed, the header is hidden completely so that only the celebratory card is visible.

<img width="1504" alt="image" src="https://user-images.githubusercontent.com/79307566/170072245-75eb4b88-9869-41a9-a727-aca8fcf36ba5.png">

When someone brings back the list after completing it, we should hide the header altogether.

Reply:
Note that in the designs the section titles have smaller fonts (14 px vs 20 px). It concerns both the task list and `Things to do next.` It's like a mobile-friendly version of the home screen.

Reply:
After taking a closer look at the design, I've applied the altered styles to task list when displayed in the sidebar in https://github.com/woocommerce/woocommerce/pull/33124/commits/e1834e3ce116cd4c8135cb81987199de8d1a6d88. 

I also ensured that the progress header is not displayed when the list is complete.

Let me know if I missed anything from this thread!